src/cljs/main/broadfcui/header.cljs

```diff
@@ -41,7 +41,8 @@
    {:label [:div {:style {:borderRadius 2
                           :backgroundColor (:background-light style/colors)
                           :color "#000" :textDecoration "none"
-                          :padding "0.5rem" :border style/standard-line}}
+                          :padding "0.5rem" :border style/standard-line}
+                  :data-test-id "accountDropdown"}
```

Comment:
I like the concision of reusing the key. I can live with camel-case if it flows naturally from the data, as it does in that case. I'd like to see language-idiomatic names when we create them.

On another project, I convert keys to kebab-case as part of the JSON parsing and conversion, and back to the server's preferred format on transmission. I've found this to be quite beneficial in making my code more internally consistent and nicer to work with. If we decide to go this route, these keys will naturally move.